src/core/reactor.cc

```diff
@@ -3748,6 +3744,11 @@ void writeable_eventfd::signal(size_t count) {
     SEASTAR_ASSERT(r == sizeof(c));
 }
 
+std::optional<size_t> writeable_eventfd::consume() {
+    uint64_t c;
+    return _fd.read(&c, sizeof(c));
+}
```

Comment:
Shouldn't this be part of the read side?

Reply:
The read-side uses await-able wrapper around pollable_fd, which is not needed by the users of this new method. Actually the writeable_eventfd and readable_eventfd are supposed to play with each other, while reactor only needs the writeable end. I'll think on it a bit more